secretmanager/regional_samples/create_update_regional_secret_label.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package regional_secretmanager
+
+// [START secretmanager_create_update_regional_secret_label]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"google.golang.org/api/option"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+// createUpdateSecretLabel updates the labels about an existing secret.
+// If the label key exists, it updates the label, otherwise it creates a new one.
+func CreateUpdateRegionalSecretLabel(w io.Writer, projectId, locationId, id string) error {
```

Comment:
issue: put function in camel case. We don't export sample functions.

Reply:
I am exporting these so that i can include these in the unit tests